lab-justin/lib/task-crud.js

```diff
@@ -0,0 +1,53 @@
+'use strict';
+
+const Task = require('../model/task');
+const debug = require('debug')('justin:task-crud');
+const AppErr = require('./app-error');
+
+const mongoose = require('mongoose');
+mongoose.Promise = require('bluebird');
+const noteCrud = require('./note-crud');
+
+debug('justin:task-crud');
+
+exports.createTask = function(reqBody){
+  return new Promise((resolve, reject) => {
+    if (!reqBody.noteId)
+      return reject(AppErr.error400('task require\'s noteId'));
+    if (!reqBody.desc)
+      return reject(AppErr.error400('tasks require\'s desc'));
+
+    reqBody.timestamp = new Date();
+    const task = new Task(reqBody);
+
+    task.save()
+    .then(task => resolve(task))
+    .catch(err => reject(err));
+  });
+};
+
+exports.fetchTaskByNoteId = function(noteId){
```

Comment:
You might want to call this 'fetchTasksByNoteId' to make it clearer that you're going to return an array of tasks, probably not a single task.